geemap/osm.py

```diff
@@ -308,13 +474,17 @@ def osm_gdf_from_geocode(
 
     Args:
         query (str | dict | list): Query string(s) or structured dict(s) to geocode.
-        which_result (INT, optional): Which geocoding result to use. if None, auto-select the first (Multi)Polygon or raise an error if OSM doesn't return one. to get the top match regardless of geometry type, set which_result=1. Defaults to None.
-        by_osmid (bool, optional): If True, handle query as an OSM ID for lookup rather than text search. Defaults to False.
+        which_result (INT, optional): Which geocoding result to use. if None,
```

Comment:
INT to lower case.
`which_result (INT, optional)` --> `which_result (int, optional)`

Reply:
Done